docs/source/tutorial.rst

```diff
@@ -325,4 +325,4 @@ And load it:
 
 .. _sample dataset: https://github.com/snipsco/snips-nlu/blob/master/samples/sample_dataset.json
 .. _sample configurations: https://github.com/snipsco/snips-nlu/blob/master/samples/configs
-.. _english one: https://github.com/snipsco/snips-nlu/blob/master/samples/configs/config_en.json
\ No newline at end of file
+.. _english one: https://github.com/snipsco/snips-nlu/blob/master/samples/configs/config_en.json
```

Comment:
Could you please discard this change? It's probably due to an automatic formatting